Dockerfile

```diff
@@ -5,11 +5,11 @@ FROM nginx:1.11.8
 RUN apt-get update \
     && apt-get install --no-install-recommends --no-install-suggests -y \
     netcat \
+    curl \
     && rm -rf /var/lib/apt/lists/*
 
 COPY wait_for_feed_server.sh /opt/
-COPY getclient /opt/
 
-HEALTHCHECK CMD /opt/getclient /tmp/certs/replicator.key /tmp/certs/replicator.crt /tmp/certs/ca.crt https://nginxproxy:5000/notifications/recent || exit 1
+HEALTHCHECK CMD curl --noproxy localhost --fail http://localhost:5000/notifications/recent || exit 1
 
-ENTRYPOINT ["/opt/wait_for_feed_server.sh","atomfeedpub","8000"]
```

Comment:
Unfortunately the reverse proxy config file (rp.conf) that written to the container image earlier in the docker file assumes a port of 8000. So an override of the atomfeedpub port would not be reached via the proxy. I did remove the actual port assignment for atomfeedpub from the docker compose file, so that container is basically private to the docker compose network now.